spynnaker/pyNN/models/neuron/synapse_types/synapse_type_exponential.py

```diff
@@ -63,8 +63,9 @@ def has_variable(self, variable):
     @overrides(AbstractSynapseType.get_values, additional_arguments={'ts'})
     def get_values(self, parameters, state_variables, vertex_slice, ts):
 
-        decay = lambda x: numpy.exp(-ts / x)  # noqa E731
-        init = lambda x: (x / ts) * (1.0 - numpy.exp(-ts / x))  # noqa E731
+        tsfloat = float(ts) / 1000.0
```

Comment:
It is probably worth checking if there is anywhere else in the changes that forgets to do this!

Reply:
I found one more in dual_exponential; I think there might be another in NewModelTemplate which I'll get to at some point...